src/modules/thirdParty/sagas.ts

```diff
@@ -153,9 +168,11 @@ export function* thirdPartySaga(builder: BuilderAPI, catalyst: CatalystClient) {
       )
 
       yield put(publishThirdPartyItemsSuccess(thirdParty.id, collectionId, newItems, newItemCurations))
-      yield put(closeModal('PublishThirdPartyCollectionModal'))
     } catch (error) {
+      yield showActionErrorToast()
```

Comment:
Should we show some specific message about the error that occurred?

Reply:
Mhmm, I don't know what else could we show to be honest, I don't think it will be useful for the user to show the BE's error.
Do you have any ideas?

Reply:
Let's go with this for the time being and if we need to get specific, we can change it.